### PR TITLE
Add translation fr_FR

### PR DESCRIPTION
**Pull request by TumbaoJu**

### CODE REVIEW

**Comment by TumbaoJu** [comment]
@alexis-via J'ai ajouté une traduction fr_FR.
Merci

File: account_invoice_overdue_reminder/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,1083 @@
...
+"% endif\n"
+"\n"
+"<p>Regards,</p>\n"
+"\n"
+"</div>\n"
+msgstr ""
+"\n"
+"<div style=\"margin: 0px; padding: 0px; font-size: 13px;\">\n"
+"<p>Cher client,</p>\n"
+"\n"
```
Comment (legalsylvain) [inline account_invoice_overdue_reminder/i18n/fr.po:85]: the translation is cutted ?

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi. Thanks for the tranlation.
it's seems that some text are not completely translated.
could you take a look ? 
thanks !

File: account_invoice_overdue_reminder/i18n/fr.po (hunk 1 above)
Comment (PADurand) [reply account_invoice_overdue_reminder/i18n/fr.po:85]: Yes it seems to have a missing part.

Comment (TumbaoJu) [reply account_invoice_overdue_reminder/i18n/fr.po:85]: @legalsylvain Yes sorry my fault.

**Comment by TumbaoJu** [comment]
I will propose the translation on OCA Weblate.

File: account_invoice_overdue_reminder/i18n/fr.po (hunk 1 above)
Comment (legalsylvain) [reply account_invoice_overdue_reminder/i18n/fr.po:85]: This module is not in the OCA. This way is thr good way !
Kind regards.

**Pull request closed without merging**